src/docs/reference/modules/Users/README.md

```diff
@@ -0,0 +1,17 @@
+# Users (OrchardCore.Users)
+
+The Users module enables authentication UI and user management.
+
+## Custom Paths
+
+If you want to specify custom paths to access the authentication related urls, you can change ithem by using this option in the appsettings.json:
+
+``` json
+  "OrchardCore": {
+    "OrchardCore_Users": {
+      "LoginPath": "Login",
+      "LogoffPath": "LogOff",
```

Comment:
the default is `Users/LogOff' if I am correct?

Reply:
Lol, half the change from you and agriffard.